src/LibTeXPrintf.jl

```diff
@@ -234,18 +243,24 @@ i=0
 
 julia> using LaTeXStrings
 
-julia> out = stexprintf(L"\\sum_{i=0}^{10}{%c}^2", 'i')
-"10\\n⎯⎯\\n╲   2\\n╱  i\\n⎺⎺\\ni=0"
+julia> out = stexprintf(L"\sum_{i=0}^{10}{%c}^2", 'i')
+" 10\n ⎯⎯\n ╲   2\n\$╱  i \$\n ⎺⎺\n i=0"
+
+julia> println(out)
+ 10
+ ⎯⎯
+ ╲   2
+$╱  i $
+ ⎺⎺
+ i=0
+
 ```
 """
-function stexprintf(format::String, args...)
-    if !iszero(length(format))
-        f = Format(format)
-        length(f.formats) == length(args) || argmismatch(length(f.formats), length(args))
-    else
-        length(args) == 0 || argmismatch(0, length(args))
-    end
-    cstr = libtexprintf.stexprintf(format, args...)
+function stexprintf(fmt::String, args...; lw=displaysize(stdout)[2])
+    str = format(Format(fmt), args...)
+    escstr = replace(str, r"%" => "%%")
```

Comment:
```suggestion
    escstr = replace(str, '%' => "%%")
```
Don't seem to need a regex here?